LoopKit/Guardrail.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  Guardrail.swift
+//  LoopKit
+//
+//  Created by Michael Pangburn on 4/10/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import HealthKit
+
+
+public enum SafetyClassification: Equatable {
+    public enum Threshold: Equatable {
+        case minimum
+        case belowRecommended
+        case aboveRecommended
+        case maximum
+    }
+
+    case withinRecommendedRange
+    case outsideRecommendedRange(Threshold)
+}
+
+public struct Guardrail<Value: Comparable> {
+    public let absoluteBounds: ClosedRange<Value>
+    public let recommendedBounds: ClosedRange<Value>
```

Comment:
OK. I think this answers my question above. I'm more used to the smart infusion pump concept of hard and soft limits instead of absolute and recommended bounds.

Reply:
Yep, same idea—confirmation required when outside recommended bounds.